test/view_models/workarea/admin/afterpay_configuration_view_model_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+module Workarea
+  module Admin
+    class AfterpayConfigurationViewModelTest < TestCase
+      def test_limits
+        Workarea::Afterpay
+          .stubs(:credentials)
+          .returns(us: { merchant_id: 'foo' }, au: { merchant_id: 'bar' })
```

Comment:
I am not seeing how this exercises the code in that this test would still pass if there were a typo in the hash keys.

Reply:
I'm not sure what you mean, the bug was mostly the call to `.first` being unnecessary. Here's the original sentry error, where it's trying to use an array instead of the hash: https://sentry.tools.weblinc.com/weblinc/workarea-v3-qa/issues/4787619/?query=is:unresolved

Removing the `.first` fixes it because it's a hash, not an array.

Reply:
Lines 28 and 33 of afterpay_configuration_view_model.rb are never reached in the test is what I am saying. So you could have totally messed up the conditional or the hash containing the values and the test would still pass.

Reply:
But I see now, that comes from them so we must assume they are sending the right things.

Reply:
True, I wasn't changing lines 28 or 33 so I didn't bother to add coverage for them.